clients/drcachesim/scheduler/scheduler.cpp

```diff
@@ -759,14 +878,36 @@ scheduler_tmpl_t<RecordType, ReaderType>::advance_region_of_interest(
 
     VPRINT(this, 2, "skipping from %" PRId64 " to %" PRId64 " instrs for ROI\n",
            cur_instr, cur_range.start_instruction);
+    if (options_.record_ostream != nullptr) {
+        sched_type_t::stream_status_t status = close_schedule_segment(output, input);
+        if (status != sched_type_t::STATUS_OK)
+            return status;
+        status = record_schedule_segment(output, schedule_record_t::SKIP, input.index,
+                                         cur_instr, cur_range.start_instruction);
```

Comment:
So we're not recording that [cur_instr, cur_range.start_instruction) was skipped?

Reply:
I don't understand: we are recording exactly that, that that range was skipped.

Reply:
I think I misread. Ignore.